packages/cactus-cmd-gui-app/README.md

```diff
@@ -0,0 +1,48 @@
+# `@hyperledger/cactus-cmd-gui-app`
+
+This component allows viewing ledger data in Supabase. The data is fed to supabase by persistence plugins for each ledgers.
+
+## Summary
+
+- [Remarks](#remarks)
+- [Getting Started](#getting-started)
+- [Contributing](#contributing)
+- [License](#license)
+- [Acknowledgments](#acknowledgments)
+
+## Remarks
+
+- Plugin requires running Supabase and persistence plugins in order to properly view ledger data.
+- Currently, only ethereum based ledgers are supported.
+
+## Getting Started
+
+Clone the git repository on your local machine. Follow these instructions that will get you a copy of the project up and running on your local machine for development and testing purposes.
```

Comment:
@ervvkb Can this not be used directly from npm via installing it to another project as a dependency? If the only way that this can be used is by copy pasting the source code of the framework then I'd call it an example rather than an actual reusable component of the framework.

If it **CAN** be used out of the box as an npm install into another project then please make sure to document how that can be done exactly as well (maybe you already did and I'm just not seeing it - sorry if this is the case)